unit-tests/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+set(fortran_src
```

Comment:
Is there a better way to point to source code in another directory?

Reply:
I found a solution by change this line from:
../sorc/chgres_cube.fd/utils.f90
to
${CMAKE_SOURCE_DIR}/sorc/chgres_cube.fd/utils.f90
I tested and it works. Is this solution to your satisfaction?